sdk/selfhelp/Azure.ResourceManager.SelfHelp/src/Generated/Models/SolutionMetadataProperties.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.ResourceManager.SelfHelp.Models
+{
+    /// <summary> Metadata Properties. </summary>
+    public partial class SolutionMetadataProperties
+    {
+        /// <summary> Initializes a new instance of SolutionMetadataProperties. </summary>
+        public SolutionMetadataProperties()
+        {
+            RequiredInputs = new ChangeTrackingList<string>();
+        }
+
+        /// <summary> Initializes a new instance of SolutionMetadataProperties. </summary>
+        /// <param name="solutionId"> Solution Id. </param>
+        /// <param name="solutionType"> Solution Type. </param>
+        /// <param name="description"> A detailed description of solution. </param>
+        /// <param name="requiredInputs"> Required parameters for invoking this particular solution. </param>
+        internal SolutionMetadataProperties(string solutionId, SolutionType? solutionType, string description, IReadOnlyList<string> requiredInputs)
+        {
+            SolutionId = solutionId;
```

Comment:
What's the format of this property value? Same question for `SolutionId` property in other models.

Reply:
As earlier mentioned, this is string